src/java.base/share/classes/java/lang/System.java

```diff
@@ -761,6 +761,9 @@ public static native void arraycopy(Object src,  int  srcPos,
      * <tr><th scope="row">{@systemProperty native.encoding}</th>
      *     <td>Character encoding name derived from the host environment and/or
      *     the user's settings. Setting this system property has no effect.</td></tr>
+     * <tr><th scope="row">{@systemProperty java.properties.date}</th>
+     *     <td>Text for the comment that must replace the default date comment
+     *     written out by {@code Properties.store()} methods <em>(optional)</em> </td></tr>
```

Comment:
To date, the table in getProperties has listed the supported system properties that the runtime makes available to applications. It hasn't historically listed the many other standard properties that can be set on the command line. So I'm sure about adding this one to the table as it opens the door to expanding the table to all the other system properties that are documented elsewhere in the API docs. Note that javadoc creates a table of system properties from usages of `@systemProperty` so there is already a more complete table in the javadoc build.

Reply:
After this change to include this property in System::getProperties() javadoc was added, there have been inputs (here and on the CSR) which suggest that we probably shouldn't include it here. I've now updated this PR to revert this specific change.